department-of-veterans-affairs/caseflow
# Add test scenarios data for Certification

Resolves #4872 

### Description
Added VACOLS_IDS to cases.csv and setup vbms fake data to create test scenarios


### Testing Plan
1. Go to /certifications/new/:VACOLS_ID 
2. Make sure data correspond to scenario
